.travis.yml

```diff
@@ -4,10 +4,12 @@ node_js:
 cache:
   directories:
     - node_modules
+before_install:
+  - npm install -g npm@latest
 before_script:
   - chmod +x scripts/docs
 script:
-  - npm run cover
+  - CI=true npm run cover
```

Comment:
It does. I couldn't find it in their docs. I was really hoping the slow tests were caused by something obvious but nothing came up. Tests run so fast locally. I think this is a better scenario than it was before but I definitely want to figure that out. I'll create an issue.